modules/3d-tiles/src/tileset/tileset-3d.js

```diff
@@ -57,7 +58,7 @@ const DEFAULT_OPTIONS = {
 
   cullWithChildrenBounds: true,
   maximumScreenSpaceError: 16,
-  maximumMemoryUsage: 512,
+  maximumMemoryUsage: 32,
```

Comment:
why reduce this `maximumMemoryUsage `?

Reply:
> why reduce this maximumMemoryUsage ?

So that we can test that cache is really working. 512 MB is very big number, bigger than entire the S:t Helens tileset.